algo.py

```diff
@@ -78,34 +79,35 @@ def sampling_iteration(self):
 
         # Read from sensors
         flow_value = self._flow_sensor.read_flow_slm() * self.ML_IN_LITER
-        pressure_value = self._pressure_sensor.read_pressure()
+        pressure_value_cmh2o = self._pressure_sensor.read_pressure() *\
+            self.PSI_CMH2O_RATIO
```

Comment:
Don't break line after math operator just to keep lines below 80.
Readability is more important than 80 characters
```suggestion
        pressure_value_cmh2o = self._pressure_sensor.read_pressure() * self.PSI_CMH2O_RATIO
```